glossary.yml

```diff
@@ -1348,6 +1348,10 @@
     def: >
       O processo de dividir os dados em grupos quando os grupos em si não são conhecidos
       antecipadamente.
+  ja:
+    term: "クラスタリング"
+    def: >
+      データをどのように分類するかは不明のまま、グループに分割するプロセスのこと。
```

Comment:
This seems a perfect translation of the English definition here in glossary.yml. However, Given the first paragraph of INTRODUCTION of ['A new approach to clustering' (1969, Ruspini)](https://doi.org/10.1016/S0019-9958(69)90591-9) on p. 23,  I feel the English definition here misses information about how the grouping is done as a result (but not "how" as means, if this makes sense). So IMO, the English definition should be something like "The process of dividing data into groups when the groups themselves are not known in advance, but so that as a result data points within a group are more similar to each other than than data points in the other groups in certain perspective."

I am not an expert of this field, so I maybe misunderstanding. I felt like the current definition feels like "grouping data in whatever way, could be random". So if this is the case, I think again the translation is perfect.

Reply:
Perhaps I can help here? Clustering is used a lot in my field.

You're right that the result of clustering is to group similar data points together but the "how" complicates things. There are many ways to perform clustering (hierarchical, graph-based, k-means, etc) so it's difficult to generalise and I think not necessarily helpful to novice users to go into detail. I agree that the English definition is very simple but I think it is okay.